sessions/test_js_challenge.py

```diff
@@ -132,14 +219,6 @@ def prepare_js_templates(self):
         remote.tempesta.run_cmd("cp %s %s/js2.tpl" % (template, workdir))
         remote.tempesta.run_cmd("cp %s %s/js3.tpl" % (template, workdir))
 
-    def start_all(self):
-        self.prepare_js_templates()
-        self.start_all_servers()
-        self.start_tempesta()
-        self.start_all_clients()
-        self.deproxy_manager.start()
-        self.assertTrue(self.wait_all_connections(1))
-
     def test_get_challenge(self):
         """Not all requests are challengeable. Tempesta sends the challenge
         only if the client can accept it, i.e. request should has GET method and
```

Comment:
Not an issue of this PR, but the comment is bit outdated, only `text/html` token enables the challenge.